src/Turbo/src/Doctrine/BroadcastListener.php

```diff
@@ -136,7 +136,7 @@ private function storeEntitiesToPublish(EntityManagerInterface $em, object $enti
                 foreach ($options as $option) {
                     $this->broadcastedClasses[$class][] = $option->newInstance()->options;
                 }
-            } elseif ($this->annotationReader && $options = $this->annotationReader->getClassAnnotations($r ?? new \ReflectionClass($class))) {
+            } elseif ($this->annotationReader && $options = $this->annotationReader->getClassAnnotations(new \ReflectionClass($class))) {
```

Comment:
Why this change?

Reply:
`$r` is never null here, it's always set in the above if statement.

Reply:
So we should only keep `$r`, isn't it?

Reply:
Hey, yes, you're right